src/main/app/src/app/shared/table-zoek-filters/facet-filter/facet-filter.component.ts

```diff
@@ -13,10 +14,10 @@ import {FilterResultaat} from '../../../zoeken/model/filter-resultaat';
 })
 export class FacetFilterComponent implements OnInit, OnChanges {
     _selected: string;
```

Comment:
Is bovenstaande underscore in _selected bewust?

Reply:
was bewust, omdat het de eerste waarde van selected was. maar heb hem weggehaald, omdat het nu de eerste waarde van filter is